### ISSUE TITLE
typing bun on win11 crashes powershell

### ISSUE THREAD

**Issue by Thinline20** [oven-sh/bun#11688]
### What version of Bun is running?

1.1.12

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

When I type only `bun`, it crashes powershell with error code 3762504530. `bun --help` and other things work fine though.

### What is the expected behavior?

it should print help command, but it crashes immediately

### What do you see instead?

```
An error has occurred that was not properly handled. Additional information is shown below. The PowerShell process will exit.
Unhandled exception. System.InvalidOperationException: Cannot read keys when either application does not have a console or when console input has been redirected. Try Console.Read.
   at System.ConsolePal.ReadKey(Boolean intercept)
   at Microsoft.PowerShell.Internal.VirtualTerminal.ReadKey()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadOneOrMoreKeys()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadKeyThreadProc()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
```

**Comment by stevebeauge** [comment]
Dup of https://github.com/oven-sh/bun/issues/10592 ?

**Comment by Jarred-Sumner** [comment]
This was fixed bun @paperdave in Bun v1.1.15 or so
